e2e/cluster_test.go

```diff
@@ -359,7 +359,8 @@ var _ = Describe("SpiceDBClusters", func() {
 					Expect(condition).To(
 						// different kube versions return different messages, this matches either
 						Or(
-							EqualCondition(v1alpha1.NewPodErrorCondition(`failed to create containerd task: failed to create shim task: OCI runtime create failed: runc create failed: unable to start container process: error during container init: exec: "badcmd": executable file not found in $PATH`)),
+							EqualCondition(v1alpha1.NewPodErrorCondition(`failed to create containerd task: failed to create shim task: OCI runtime create failed: runc create failed: unable to start container process: error during container init: exec: "badcmd": executable file not found in $PATH: unknown`)),
+							EqualCondition(v1alpha1.NewPodErrorCondition(`failed to create containerd task: failed to create shim task: OCI runtime create failed: runc create failed: unable to start container process: exec: "badcmd": executable file not found in $PATH: unknown`)),
```

Comment:
```
EqualCondition(v1alpha1.NewPodErrorCondition(`failed to create containerd task: failed to create shim task: OCI runtime create failed: runc create failed: unable to start container process: exec: "badcmd": executable file not found in $PATH`)),
```
?

Reply:
This looks like it's changing one of the existing ones instead of just adding a new one